crates/holochain/src/conductor/conductor.rs

```diff
@@ -3299,6 +3304,87 @@ mod countersigning_impls {
                 .await?,
             )
         }
+
+        /// Get in-memory state of an ongoing countersigning session.
+        pub async fn get_countersigning_session_state(
+            &self,
+            cell_id: &CellId,
+        ) -> ConductorResult<Option<CountersigningSessionState>> {
+            let space = self.get_or_create_space(cell_id.dna_hash())?;
+            let maybe_countersigning_workspace =
+                space.countersigning_workspaces.lock().get(cell_id).cloned();
+            match maybe_countersigning_workspace {
+                None => Ok(None),
```

Comment:
Log here to help people who've called the API with a wrong cell id? The workspace should always have been created for the cell so if it's missing here then the cell id is probably invalid

Reply:
Good point. Actually it should return an error, because it's theoretically impossible for there not to be a workspace for a valid cell, so the cell id must be invalid. I've updated the docs and extended the error to reflect that.